### DISCUSSION TITLE
Parametric initial conditions for time-dependent PDE

### DISCUSSION THREAD

**Discussion by JeanYvesDieulot** [lululxvi/deepxde#1686] [Q&A]
Hi,
I started representing a chemical reactor, with 2 species; the model consists of two time dependent 1-D PDEs.
I managed, thanks to the library, to have nice results with fixed BC and fixed ICs. For the moment, the ICs are zero, that is C1(x,0)=0, C2(x,0)=0 , C1 and C2 being the output variables.
I understand that if I want to parametrize the PINN (e.g. with reactor dimension), then I should augment the input vector. Now, I would like to know how I can parametrize the PINN using "any" ICs. Do I have to interpolate the ICs by parametric functions and pass their parameters to the PINN, or is there another solution?
Thanks in advance
Jean-Yves

**Comment by lululxvi** [comment]
Use DeepONet.
